## 50_disable_epn by Jan Wagner <waja@cyconet.org>
##
## DP: leave marks to disable epn

--- a/plugins/check_snmp_boostedge.pl
+++ b/plugins/check_snmp_boostedge.pl
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_boostedge.pl #################
 # Version : 1.0
 # Date : Jan 16 2007
--- a/plugins/check_snmp_cpfw.pl
+++ b/plugins/check_snmp_cpfw.pl
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_cpfw ##############
 my $VERSION = "2.0.0";
 
--- a/plugins/check_snmp_css_main.pl
+++ b/plugins/check_snmp_css_main.pl
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_css_main.pl #################
 # Version : 1.0
 # Date : 27 Sept 2006
--- a/plugins/check_snmp_css.pl
+++ b/plugins/check_snmp_css.pl
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_css.pl #################
 # Version : 1.0.1
 # Date : 27 Sept 2006
--- a/plugins/check_snmp_env.pl
+++ b/plugins/check_snmp_env.pl
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_env #################
 # Version : 1.3
 # Date : May 24 2007
--- a/plugins/check_snmp_int.pl
+++ b/plugins/check_snmp_int.pl
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w
+# nagios: -epn
 ############################## check_snmp_int ##############
 my $VERSION = "2.0.0";
 
--- a/plugins/check_snmp_linkproof_nhr.pl
+++ b/plugins/check_snmp_linkproof_nhr.pl
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_linkproof_nhr #################
 # Version : 1.0
 # Date : Aug 24 2006
--- a/plugins/check_snmp_load.pl
+++ b/plugins/check_snmp_load.pl
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_load #################
 my $VERSION = "2.0.0";
 
--- a/plugins/check_snmp_mem.pl
+++ b/plugins/check_snmp_mem.pl
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_mem ##############
 my $VERSION = "2.0.0";
 
--- a/plugins/check_snmp_nsbox.pl
+++ b/plugins/check_snmp_nsbox.pl
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_nsbox #################
 # Version : 1.0
 # Date : Jan 16 2007
--- a/plugins/check_snmp_process.pl
+++ b/plugins/check_snmp_process.pl
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w
+# nagios: -epn
 ############################## check_snmp_process ##############
 my $VERSION = "2.0.0";
 
--- a/plugins/check_snmp_storage.pl
+++ b/plugins/check_snmp_storage.pl
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w
+# nagios: -epn
 ############################## check_snmp_storage ##############
 # Version : 1.3.3
 # Date :  Jun 1 2007
--- a/plugins/check_snmp_vrrp.pl
+++ b/plugins/check_snmp_vrrp.pl
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w 
+# nagios: -epn
 ############################## check_snmp_vrrp ##############
 my $VERSION = "2.0.0";
 
--- a/plugins/check_snmp_win.pl
+++ b/plugins/check_snmp_win.pl
@@ -1,4 +1,5 @@
 #!/usr/bin/perl -w
+# nagios: -epn
 ############################## check_snmp_win ##############
 my $VERSION = "2.0.0";
 
